de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/dvlm/categories/propertyinstances/provider/DVLMValuePropertyInstanceItemProvider.java

```diff
@@ -39,6 +42,13 @@ public String getText(Object object) {
 		if (object instanceof ValuePropertyInstance) {
 			ValuePropertyInstance vpi = (ValuePropertyInstance) object;
 			String value = vpi.getValue() == null ? "" : vpi.getValue();
+
+			//If string is multiline then just show the first line
+			String lineSeperator = System.getProperty(LINE_SEPERATOR_PROPERTY);

```

Comment:
Is there a reason we cant use System.lineSeparator() to get the property value directly?

Reply:
Didn't know about that property yet